src/renderer/entities/governance/lib/governanceSubscribeService.ts

```diff
@@ -138,3 +144,24 @@ function subscribeVotingFor(
     unsubscribe.then((fn) => fn());
   };
 }
+
+/**
+ * TODO move to chain helpers
+ */
+function subscribeBlockWithInterval(api: ApiPromise, ttl: number, callback: (header: Header) => unknown) {
+  const throttled = throttle(callback, ttl);
+  const unsubscribe = api.rpc.chain.subscribeNewHeads((header) => {
+    throttled(header);
```

Comment:
Trying to understand, why do we need to pass `header` into `throttled`?

Reply:
It can be useful, when this helper became shared